packaging/debian/datadog-agent.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description="Datadog Agent"
+After=network.target
+
+[Service]
+Type=forking
+Environment=SYSTEM_PATH=/opt/datadog-agent/embedded/bin:/opt/datadog-agent/bin:$PATH
+ExecStart=/bin/bash -c "PATH=${SYSTEM_PATH} /opt/datadog-agent/bin/supervisord -c /etc/dd-agent/supervisor.conf --pidfile /var/run/datadog-supervisord.pid"
+ExecReload=/opt/datadog-agent/bin/supervisorctl reload
```

Comment:
I actually made it. I just have to push it has soon has I am able to run omnibus again and test the built packages.